src/rust/sdk/example/src/main.rs

```diff
@@ -0,0 +1,353 @@
+    let tick_size = 10u64.pow(6 - 3); // eAPT has 6 decimals, want 1/1000th granularity
+    let min_size = 1;
+
+    let entry = register_market_base_coin_from_coinstore(econia_address, &e_apt, &e_usdc, &APTOS_COIN_TYPE, lot_size, tick_size, min_size).unwrap();
+
+    econia_client.submit_tx(entry).await?;
+
+    let market_id = econia_client.view_client().get_market_id_base_coin(e_apt.clone().into(), e_usdc.clone().into(), lot_size, tick_size, min_size).await?;
+
+    let market_id = if let Some(market_id) = market_id {
+        println!("Market created with ID: {market_id}");
+        market_id
+    } else {
+        panic!("Could not create market."); // Should not happen
+    };
+
+    wait_for_enter!();
+
+
+    print_title!("Set up account A");
+
+    let (account_address_a, mut econia_client_a) = account(&faucet_client, &args.url, econia_address.clone()).await;
+
+    fund(&e_apt, 10u64.pow(18), &mut econia_client_a, faucet_address).await?;
+    println!("Minted EAPT to {account_address_a}");
+
+    fund(&e_usdc, 10u64.pow(10), &mut econia_client_a, faucet_address).await?;
+    println!("Minted EUSDC to {account_address_a}");
+
+    let entry = register_market_account(econia_address, &e_apt, &e_usdc, market_id, 0)?;
+    econia_client_a.submit_tx(entry).await?;
+    println!("Registered market account for {account_address_a}");
+
+    let e_apt_subunits = 10u64.pow(9);
+    let entry = deposit_from_coinstore(econia_address, &e_apt, market_id, 0, e_apt_subunits)?;
+    econia_client_a.submit_tx(entry).await?;
+    println!("Deposited EAPT from coinstore for account {account_address_a}");
+
+    let e_usdc_subunits = 10u64.pow(10);
+    let entry = deposit_from_coinstore(econia_address, &e_usdc, market_id, 0, e_usdc_subunits)?;
+    econia_client_a.submit_tx(entry).await?;
+    println!("Deposited EUSDC from coinstore for account {account_address_a}");
+
+    println!("{account_address_a} was successfully set up");
+
+    wait_for_enter!();
+
+
+    print_title!("Place two limit orders with account A");
+
+    let buy_base_lots = 10u64.pow(3);
+    let buy_ticks_per_lot = 10u64.pow(3);
+    let entry = place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Bid, buy_base_lots, buy_ticks_per_lot, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?;
+    econia_client_a.submit_tx(entry).await?;
+    println!("Placed bid order for account {account_address_a}");
+
+    let sell_base_lots = 10u64.pow(3);
+    let sell_ticks_per_lot = 2 * 10u64.pow(3);
+    let entry = place_limit_order_user_entry(econia_address, &e_apt, &e_usdc, market_id, &econia_address, Side::Ask, sell_base_lots, sell_ticks_per_lot, Restriction::NoRestriction, SelfMatchBehavior::CancelMaker)?;
+    econia_client_a.submit_tx(entry).await?;
+    println!("Placed ask order for account {account_address_a}");
+
+    wait_for_enter!();
+
+
+    print_title!("Set up account B");
```

Comment:
It would be messy. It would take like 6 args and return 2 elements. I think it's ok to do it this way. Plus it's easier to go through. I tried to make functions only for things not related to Econia. Econia related stuff should (IMO) be in the `main` function, easier to go through when following the tutorial than to jump from function to function